docs/en_US/Tutorial/HowToUseSharedStorage.rst

```diff
@@ -0,0 +1,50 @@
+**How to Use Shared Storage**
```

Comment:
Please explicitly list supported distros and state that it requires root privilege.

Reply:
add a note below.